packages/gatsby-plugin-benchmark-reporting/src/gatsby-node.js

```diff
@@ -272,6 +275,21 @@ class BenchMeta {
   }
 }
 
+function init(lifecycle) {
+  if (!benchMeta) {
+    benchMeta = new BenchMeta()
+    // This should be set in the gatsby-config of the site when enabling this plugin
+    reportInfo(
+      `gatsby-plugin-benchmark-reporting: Will post benchmark data to: ${
+        BENCHMARK_REPORTING_URL || `the CLI`
+      }`
+    )
+
+    benchMeta.markStart()
+    benchMeta.markDataPoint(lifecycle)
```

Comment:
Doesn't this make it call any lifecycle twice for markDataPoint when it hasn't started?

Reply:
Oh dang, you're right. Removed it